cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/NewCommand.java

```diff
@@ -53,9 +53,9 @@
     @CommandLine.Option(names = {"--help", "-h"}, hidden = true)
     private boolean helpFlag;
 
-    @CommandLine.Option(names = {"--template", "-t"}, description = "Acceptable values: [main, service, lib] " +
-            "default: main")
-    private String template = "main";
+    @CommandLine.Option(names = {"--template", "-t"}, description = "Acceptable values: [general, main, service, lib]" +
```

Comment:
A default is more appropriate name for the default template

Reply:
IMO, we do not need to show it to the user. We can remove it from the command description